File: snippets/fsharp/VS_Snippets_CLR/DateTime.ToFileTime/FS/class1.fs
//<Snippet1>
open System.IO

printfn "Enter the file path:"
let filePath = stdin.ReadLine()

if File.Exists filePath then
    let fileCreationDateTime =
        File.GetCreationTime filePath

    let fileCreationFileTime = fileCreationDateTime.ToFileTime()

    printfn $"{fileCreationDateTime} in file time is {fileCreationFileTime}."

else
    printfn $"{filePath} is an invalid file"

//</Snippet1>

File: snippets/csharp/System/DateTime/ToFileTime/class1.cs


namespace ToFileTime
{
    class Class1
    {
        //<Snippet1>
        static void Main(string[] args)
        {
            System.Console.WriteLine("Enter the file path:");
            string filePath = System.Console.ReadLine();

            if (System.IO.File.Exists(filePath))
            {
                System.DateTime fileCreationDateTime =
                    System.IO.File.GetCreationTime(filePath);

                long fileCreationFileTime = fileCreationDateTime.ToFileTime();

                System.Console.WriteLine($"{fileCreationDateTime} in file time is {fileCreationFileTime}.");
            }
            else
            {
                System.Console.WriteLine($"{filePath} is an invalid file");
            }
        }
        //</Snippet1>
    }
}
